frontend/www/js/omegaup/time.ts

```diff
@@ -120,22 +121,21 @@ export function parseDateTimeLocal(dateString: string): Date {
   // The expected format is yyyy-MM-ddTHH:MM in the local timezone.
   // Date.parse() will use UTC if given a timestamp with that format, instead
   // of the local timezone.
-  const result = new Date();
+  const result = new Date(dateString);
```

Comment:
```suggestion
  const result = new Date();
```
según yo nomás falta esto y ya!

Reply:
Listo.